cli/command/swarm/opts.go

```diff
@@ -24,13 +24,17 @@ const (
 	flagToken               = "token"
 	flagTaskHistoryLimit    = "task-history-limit"
 	flagExternalCA          = "external-ca"
+	flagKeepOldSnapshots    = "keep-old-snapshots"
```

Comment:
I think that would work; is the "current" version considered a "snapshot"? Not an expert in this area, but "snapshot" feels as "previous versions", so 0 being "no previous versions", thus "no snapshots", works for me